.travis.yml

```diff
@@ -27,3 +27,6 @@ env:
   - secure: kxnTz51/kPI6y4vU8y222rX1B0csVWXU3pvJVzKxgBuoOtvSsebqsjPXkBT67j0anf7yg/N9qIFXf/8J2QWOAs3mYS0viONppGsiAUqla9QwpEJG89equl/760J5RIiSswsMTI/Rd1CM53AF7lUudF5+htXt9VkWvOAbJzkCTAdsqJZgNIQYDezkRxEwpmaIVZNzh/wsjW7x8as4sAeq3qZeVGRA6skEJiOIoR780EFOQqVz+BH+17NreFFM60ekkLBTzjCjkRyWCZmNl8FMz/F/E3qN/21y6ggqfBJf9qGO9GJ3xAr5FhGmyllMP4xKyQeBlUkbzJHWFnAvJFOQ+4dlVech9X3RFczQwMpnVT/m6rQyYy7xQXw+2f6iiAXf4biPsWJgEOBuJwgNIIGAWTbxvH7k5xa5iDbKFntw0TIzAopNZI6JBd5kUQHri0544q2akHRZzcpa+0ZBqOiqMMoL3pkws2L8E8VYKofL3vmP/2o298Ah0KYZWsl6Oc5Ev/vfQLn+cGROpgoeoURW601kFpJ4+uLNhQwdxavEI0KnRLK9OZKik0mHzy/LX/xp6aKl1jREc04pONzSHqQtkhSLmjauQoF0e6UXWCPdH4LW1bd1Xw0BhvyOjSiy2gOyRfj9WHunn9pb4oDsYQbswDIlew6wQR6uKoc7d7/rUDI=
   - secure: h4pyX3Ak/ZpKyMWUKOcdng6Psq1FqAsTG0ErXUwxWQ16ijHZZL5jlcMnDpH9ctRjyaS8s/3dfa2dRAuPCt9cQOBXZBWNCGekr5kpVq8eaYDJ1SOzEvm8CPtNkqof46ifYN+KDFtqodg83M9Mm2ab0MZgvCP7JJd072SnafhQvXyZ4rOucocnbAp31BFtHhMLlGpI+Vr/1jl88Df1tHDyiPdVjqzNm0z9CGttFkmOpO9Uah+6hKKslDZGtFiRxnf4rpiWSYKkkWe9pa1330V37Cfmz8T4k4c8irg2Hi3ShojAmgRNuwq6Dg8skzy61R172FWpb6KhmT/usRX15FQ005gkM4NXJ3oRNORgsjiUtCKHDFlCJiFO/suqIcWxvEPvQjBJgAL+efxxku2S9wUGoLk3mw2KCHuw+Q/tj+kaqtO5XbtUH3h43M6UqC37E+ZLMnQnQx4I7YcsfCY54pwxbwgRDm/k/5Egs596fKxUJO4VBY04FdB1tqxleVG1GN3fv6qfcZaUDj3VPEQ1d5jFzHcGm5QVEbsyfh/FSdgUmMTuWtC8orqIZ8Rt774HJjzjUw9SkiBCQMk5dI5vi8vMbf3JHQVqQW+LdDW+Z5qd4ZlVYqPOK/FBspq54qpIFIA+6mmmUmzJOzpmEokNhoDq3gBGBDD+gqXJpQ6Vi5qe1/E=
   - secure: R5a9OOHyki7lihwtbWuWaIBv7nwKY8lCGRmYaSiZ5rKZ5ZSkaoC55eq3JsFqW1sQTxsiri6UJKBXoxJJiY1veV5DVuER6ZRoYd3Vr55BjKMBBFSp5uTFrzX7Mtbfncuf5/tVbwoDxDgc/AwahbWvR/gV9yUDGrbMrgU1/jE0Otjw+LXEqc0+nINoMVyfYFT9Elt1zcl6iKIKS+LDO0S9B5jQpxyIudC2bh/k8uaSKYRaBG9srqMuPJKXOdcMxmtOLkjcr7pKT8+LL7Y6U9i/tpsYnXHv/hz6m084Wz/Eu634wPjEfOTExoXgH1frxGF24ZgmdSfo6xm3fjI+1FPyyB7kp+y4r/pyySjHtqWJgIIUR1no22ISA0z6vy/mtQ0IXPgqoXoF96F2U2hyqVp9Mu5IXASiJXGvafW/bq4cstwRNibC8llmT5QJwJFZLs47qwCroLnyapr8Zbx6+4vzqTX6jQDURy8b95ydewtMhmwLeTeOhX+3qWnYqkG/+thhAkk1bjnuZkFgLMx9ae7GaZJSXelxhLR3pSpr9yGQ/WGnWjeumk3bN/gM8c94VCv2WDFBqb4bOjYsHZk+gGkh4mr1MznrjHcf6K7QJf9eVOXGwq4F/AMAqNkYcx3qj02b2PRrZ1WvF4MPGOw2PHREBLUzIiJmYl/iopT76XBo+f8=
+  matrix:
+  - ''
```

Comment:
ahhhhhh, okay, that definitely needs a comment or even better a `USE_FIREFOX_SCHEMAS=` (empty) or so.